lib/cc/services/github_pull_requests.rb

```diff
@@ -72,19 +57,27 @@ def simple_failure(message)
     { ok: false, message: message }
   end
 
+  def response
+    @response || simple_failure("Nothing happened")
+  end
+
   def update_status_skipped
-    update_status("success", "Code Climate has skipped analysis of this commit.")
+    update_status(
+      "success",
+      "Code Climate has skipped analysis of this commit."
+    )
   end
 
   def update_status_success
     update_status("success", "Code Climate has analyzed this pull request.")
+    add_comment
```

Comment:
Right. This class is doing too much.

I'm fine with the swap, but that means we're actually change the functionality of this (and its return value). Currently, [we return the status of add comment](https://github.com/codeclimate/codeclimate-services/blob/4aa7dfaa1a09438e41c27afa4bcebbe8ea3c9e58/lib/cc/services/github_pull_requests.rb#L49-L50) only. Do you think we should swap it anyway?